### PR TITLE
connlist output doc update

### PR DESCRIPTION
**Pull request by shireenf-ibm**
#282

### CODE REVIEW

**Review by adisos — CHANGES_REQUESTED** [review]

File: docs/connlist_output.md
[the remark this answers is not in the document]
Comment (shireenf-ibm) [reply docs/connlist_output.md:1]: sure, shouldn't it be for `netpol-analysis-example-minimal` (the diff example is diff from this)/

[the remark this answers is not in the document]
Comment (adisos) [reply docs/connlist_output.md:1]: right, yes

[the remark this answers is not in the document]
Comment (shireenf-ibm) [reply docs/connlist_output.md:1]: done

File: docs/connlist_output.md [hunk 1]
```diff
@@ -15,30 +15,88 @@ Its connectivity lines are of the form: `{ingress-controller}` => `dst` : `conne
...
+	"0.0.0.0-255.255.255.255" -> "default/frontend[Deployment]" [label="TCP 8080" color="gold2" fontcolor="darkgreen"]
+	"default/frontend[Deployment]" -> "0.0.0.0-255.255.255.255" [label="UDP 53" color="gold2" fontcolor="darkgreen"]
+	"default/frontend[Deployment]" -> "default/backend[Deployment]" [label="TCP 9090" color="gold2" fontcolor="darkgreen"]
+}
+```
+
+`svg` graph from `dot` format output can be produced using `graphviz` as following:
+```
+dot -Tsvg test_outputs/connlist/netpol-analysis-example-minimal_connlist_output.dot -o test_outputs/connlist/netpol-analysis-example-minimal_connlist_output.dot.svg
+```
```
Comment (adisos) [inline docs/connlist_output.md:94]:
```suggestion
dot -Tsvg test_outputs/connlist/netpol-analysis-example-minimal_connlist_output.dot -O
```
```

**Review by adisos — APPROVED** [review]

**Pull request merged**